Document the Attach node (#1126)
gravwell/wiki: flows/nodes/attach.md

# Attach

> The Attach node connects to an existing Gravwell query, specified by a search ID. Like the [Run Query](runquery) node, it outputs a structure into the payload (named `search` by default) which contains information about the search and allows other nodes to access the results.

Reviewer: ```suggestion
The Attach node connects to an existing Gravwell query, specified by a search ID. Like the [Run Query](runquery) node, it outputs a structure (named `search` by default) containing information about the search into the payload and allows other nodes to access the results.
```
Same comment below... I think "containing information about the search" describes the structure...?
A participial phrase should be directly next to the noun it modifies.

> The most common use of the Attach node is to connect to the scheduled search which triggered the [Alert](/alerts/alerts) for which the current flow is a consumer. See the example below for more information on this use-case.

Reviewer: ```suggestion
The most common use of the Attach node is to connect to the scheduled search which triggered the [Alert](/alerts/alerts) for which the current flow is a consumer. See the example below for more information on this use case.
```

I don't think we need a hyphen here and I don't think I typically see it hyphenated on other sites.

## Configuration

* `Search ID`, required: the ID of an existing Gravwell query.
* `Output Variable Name`: the name to use for results in the payload, default "search".

## Output

> The node inserts an object (named `search` by default) into the payload containing information about the search. The component fields are documented in the [Run Query node's documentation](runquery).

Reviewer: ```suggestion
The node inserts an object (named `search` by default) containing information about the search into the payload. The component fields are documented in the [Run Query node's documentation](runquery).
```

I think "containing information about the search" describes the object rather than the payload...?
A participial phrase should be directly next to the noun it modifies.
